File: APICuponSmart/src/java/modelo/pojo/respuesta/RespuestaCliente.java
package modelo.pojo.respuesta;

import java.util.List;
import modelo.pojo.entidad.Cliente;

public class RespuestaCliente extends Mensaje{
    private List<Cliente> contenido;
    
    public List<Cliente> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Cliente> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaCliente.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Cliente

class RespuestaCliente(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Cliente>? = null
): Mensaje(error, mensaje)